handler/github.go

```diff
@@ -72,7 +72,8 @@ func ParseNumPages(resp *github.Response) int {
 	return ParseNumPagesFromLink(link)
 }
 
-func GetPullRequests(ctx context.Context, client *github.Client, owner string, repo string, opts *github.ListOptions) ([]*github.PullRequest, error) {
+// GetPullRequests fetches all of the open pull requests from the specified repository.
```

Comment:
This comment is not necessary. We try to add code comments to a minimum - the implementation should be self-explanatory.

Reply:
Albeit unrelated with this pull request, could you remove the PaginatedRequest function from this file and use the one in https://github.com/reviewpad/reviewpad/blob/main/utils/github.go#L39?